Vision/Vision.bs

```diff
@@ -114,7 +114,9 @@ Markup Shorthands: markdown yes
 
 	These core values will be clearly demonstrated by how W3C leads the Web forward: 
 	by being inclusive, principled, and continually striving to make the Web better 
-	through these principles and the Ethical Web Principles. 
+	through these principles and the Ethical Web Principles.
+	This will lead to a Web that is more equitable,
```

Comment:
```suggestion
	This will lead to a Web that is more equitable and
```

Reply:
...just a suggestion to make this scan a bit better...

Reply:
then you would need to change the next clause to match, e.g. "better serves its..."

I don't have a preference for one framing or the other.

Reply:
I agree with @torgo I think 

> This will lead to a web that is more equitable and better serves it's users in connecting the world.

reads a little easier.

Reply:
> I agree with @torgo I think
> 
> > This will lead to a web that is more equitable and better serves it's users in connecting the world.
> 
> reads a little easier.

"its" (no apostrophe in the possessive), please